iso.c

```diff
@@ -62,7 +63,7 @@ iso_send_connection_request(char *username, uint32 neg_proto)
 	STREAM s;
 	int length = 30 + strlen(username);
 
-	if (g_rdp_version >= RDP_V5 && g_negotiate_rdp_protocol)
+	if (g_rdp_version >= RDP_V5 )
```

Comment:
I think that you should not change this line in this PR.

Reply:
if g_rdp_version >= RDP_V5, we  still have to have to negotiate protocol to have the extended_data reply ,